container-scan/action.yaml

```diff
@@ -15,50 +27,88 @@ inputs:
   lw-access-token:
     required: true
     description: Lacework access token
+  image-platform:
+    description: Target platform to build image for (eg. linux/amd64 (default), linux/arm64, etc)
+    required: false
+    default: linux/amd64
   image-tag:
     required: true
     description: Docker image tag
+  aws-access-key-id:
+    required: false
+    default: ""
+    description: AWS access key id
+  aws-secret-access-key:
+    required: false
+    default: ""
+    description: AWS secret access key
+  aws-region:
+    required: false
+    default: "us-east-1"
+    description: AWS region
```

Comment:
Why are these values needed? It seems these are values required by a service. It doesn't seem this would scale if a new service needs to pass X or Y secret / build arg. We should probably think of a better mechanism to allow a consumer to pass any secret / build arg